capstone/src/main/webapp/scripts/post.js

```diff
@@ -1,21 +1,18 @@
 let postResponse;
 
 function loadPosts() {
-  fetch("/group-post")
-    .then(response => response.json())
-    .then(postsResp => {
-      postResponse = postsResp;
-      const posts = postResponse["posts"];
-      const allPostsList = document.getElementById("posts-container");
-      allPostsList.innerHTML = "";
-      for (let i = 0; i < posts.length; i++) {
-        allPostsList.appendChild(
-          createSinglePost(posts[i], postResponse["likedPosts"])
-        );
-      }
-      addCommentInputListener();
-      addLikeButtonListener();
-    });
+  console.log(groupId);
+  fetch(`/group-post?groupId=${groupId}`).then(response => response.json()).then((postsResp) => {
+    postResponse = postsResp;
```

Comment:
You shouldn't need this line, you should be able to change `(postsResp) =>` to `(postResponse) =>`